### PR TITLE
Docs update: quarkus/spring instead of WildFly/Tomcat, CS instead of DRL, workbench/ES is dead, ...

### PR DESCRIPTION
**Pull request by ge0ffrey**

### CODE REVIEW

File: content/localized/ja/UserGuide-chapter1.adoc [hunk 1]
```diff
@@ -1,393 +0,0 @@
-= Chapter 1. OptaPlanner イントロダクション
```
Comment (ge0ffrey) [inline content/localized/ja/UserGuide-chapter1.adoc:1]: This Japanese user guide chapter 1 was hopelessly stale (Still pointing to version 6.4 images). It's better to not have it all.

**Review by triceo — CHANGES_REQUESTED** [review]
Please remove BAVET from the pictures. It lacks features to make it work with the vast majority of our exampes, and introduces the user to a choice that they really don't need to be making.

**Comment by ge0ffrey** [comment]
@triceo In that architecture picture, Bavet needs to be there, as it shows how Drools relates to ConstraintStreams, there being a second implementation, and there needs to be an illustration how all that fits together. In all the text, we only mention Drools.

How about I add "(experimental)" next to Bavet?

**Comment by triceo** [comment]
Actually, I think you don't need to mention Drools in terms of CS at all. It is an implementation detail, no need to share that with people. Especially for new folks - what is this Drools you speak of, why do I need to start looking it up?

**Comment by ge0ffrey** [comment]
@triceo Bavet removed, should be ok to merge now

**Review by triceo — APPROVED** [review]

**Pull request merged**